app/models/notification.rb

```diff
@@ -0,0 +1,10 @@
+class Notification < ActiveRecord::Base
+  belongs_to :user
+  belongs_to :source, polymorphic: true
+
+  class << self
+    def unread_for_user(id)
```

Comment:
I would then have to reference the User object in the method. Is that acceptable? Feels kind of strange to me